packages/platform/atoms/availability/AvailabilitySettings.tsx

```diff
@@ -291,11 +292,32 @@ export function AvailabilitySettings({
         schedule ? (
           schedule.schedule
             .filter((availability) => !!availability.days.length)
-            .map((availability) => (
-              <span
-                key={availability.startTime.valueOf().toString()}
-                className={cn(customClassNames?.subtitlesClassName)}>
-                {availabilityAsString(availability, { locale: i18n.language, hour12: timeFormat === 12 })}
+            .map((availability) =>
+              availabilityAsString(availability, {
+                locale: i18n.language,
+                hour12: timeFormat === 12,
+              })
+            )
+            // sort the availability strings as per user's weekstart (settings)
+            .sort((a, b) => {
+              const weekNames = weekdayNames(
+                i18n.language,
+                ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"].indexOf(
+                  weekStart || "Sunday"
+                ) as 0 | 1 | 2 | 3 | 4 | 5 | 6,
+                "short"
+              );
+              const weekIndex = (day: string) => {
+                for (let i = 0; i < weekNames.length; i++) {
+                  if (day.includes(weekNames[i])) return i;
+                }
+                return -1;
```

Comment:
Same as @surajgjadhav suggested, make it reusable.

Reply:
done